.eslintrc

```diff
@@ -1,11 +1,9 @@
 {
   "extends": [
-    "airbnb-typescript/base",
-    "prettier",
     "plugin:import/recommended",
-    "plugin:import/typescript"
+    "plugin:import/typescript",
+    "plugin:prettier/recommended"
```

Comment:
From https://github.com/prettier/eslint-plugin-prettier#configuration-legacy-eslintrc:

> This will:
> 
> - Enable the prettier/prettier rule.
> - Disable the arrow-body-style and prefer-arrow-callback rules which are problematic with this plugin - see the below for why.
> - Enable the eslint-config-prettier config which will turn off ESLint rules that conflict with Prettier.